src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/list/AdminAreasOfLawEndpointTest.java

```diff
@@ -31,6 +32,8 @@
     private static final String TEST_EXTERNAL_LINK_DESC = "Information about evicting tenants";
     private static final String TEST_EXTERNAL_LINK_DESC_CY = "Gwybodaeth ynglŷn â troi tenantiaid allan";
     private static final String TEST_DISPLAY_EXTERNAL_LINK = null;
+    private static final int AREA_OF_LAW_NEW_TEST_ID = 34_258;
+    private static final String AREA_OF_LAW_NEW_TEST_NAME = "TEST";
```

Comment:
Super small, so up to you. May make it clearer to call this one `TEST_NAME`, and the one above from `TEST_NAME` to `HOUSING_NAME`. Maybe! 

Only thought behind this was the NEW_TEST_NAME part, being similar to the TEST_NAME variable above